Sources/StreamChatSwiftUI/ChatChannelList/SearchResultsView.swift

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+struct SearchResultsView<Factory: ViewFactory>: View {
+    
+    @Injected(\.colors) private var colors
+    
+    var factory: Factory
+    @Binding var selectedChannel: ChannelSelectionInfo?
+    var searchResults: [ChannelSelectionInfo]
+    var onlineIndicatorShown: (ChatChannel) -> Bool
+    var channelNaming: (ChatChannel) -> String
+    var imageLoader: (ChatChannel) -> UIImage
+    var onSearchResultTap: (ChannelSelectionInfo) -> Void
+    var onItemAppear: (Int) -> Void
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            HStack {
+                Text("\(searchResults.count) results")
+                    .foregroundColor(Color(colors.textLowEmphasis))
```

Comment:
Are we sure contrast is high enough with gray on gray here?

Reply:
It's tricky since the placeholder color can't be changed. Will check again.